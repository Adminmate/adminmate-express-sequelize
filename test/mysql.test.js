// see https://stackoverflow.com/questions/46227783/encoding-not-recognized-in-jest-js
require('mysql2/node_modules/iconv-lite').encodingExists('foo');

process.env.DIALECT = 'mysql';

// Init app
require('./app.js');

require('./tests/model-query.test.js');