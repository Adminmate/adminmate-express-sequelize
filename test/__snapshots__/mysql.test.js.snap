// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart PIE - Count 1`] = `
Object {
  "data": Array [
    Object {
      "key": 4,
      "value": 1,
    },
    Object {
      "key": 5,
      "value": 1,
    },
  ],
}
`;
